From 5d242ae90c8efae8d12ee785a69175dd8ddfe920 Mon Sep 17 00:00:00 2001
From: Harshitha Prem <quic_hprem@quicinc.com>
Date: Fri, 1 Dec 2023 13:38:51 +0530
Subject: [PATCH] wpa_supplicant: add 320MHz bandwidth support for config

Currently, in wpa_supplicant only upto 80+80 MHz is supported
for max_oper_chwidth for config files but actually we can config
upto 320 MHz and it is handled inline.

Add code changes to support till 320 MHz for max_oper_chwidth.

Signed-off-by: Harshitha Prem <quic_hprem@quicinc.com>
---
 wpa_supplicant/config.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/wpa_supplicant/config.c
+++ b/wpa_supplicant/config.c
@@ -2638,7 +2638,7 @@ static const struct parse_data ssid_fiel
 	{ INT_RANGE(he, 0, 1) },
 	{ INT_RANGE(ht40, -1, 1) },
 	{ INT_RANGE(max_oper_chwidth, CONF_OPER_CHWIDTH_USE_HT,
-		    CONF_OPER_CHWIDTH_80P80MHZ) },
+		    CONF_OPER_CHWIDTH_320MHZ) },
 	{ INT(vht_center_freq1) },
 	{ INT(vht_center_freq2) },
 #ifdef IEEE8021X_EAPOL
